main.js

```diff
@@ -8,6 +8,13 @@ import {
 
 import modelHandler from './popup.js';
 
```

Comment:
# 💾 💾 💽 [OPTIONAL]   💽 💾 💾
**✔️ _Keep professionalism in your Repo's_** 

You can improve your repositories management:
- Create a assets directory and save all your content there.

![Screen Shot 2021-11-10 at 17 33 08](https://user-images.githubusercontent.com/22258274/141210438-fcc1991b-72f2-4ee6-a01a-11f2612c21f0.png)